# Analysis Notebook

In [11]:
# Importing all the libraries necessary for this Project

import pandas as pd
import numpy as np
import math
import matplotlib.pyplot as plt
import seaborn as sns

In [12]:
# importing dataset

Sales = pd.read_csv("Cleaned_Sales.csv")

In [91]:
Sales.head(60)

Unnamed: 0  Rank                                          Name Platform  \
0            0     1                                    Wii Sports      Wii   
1            1     2                             Super Mario Bros.      NES   
2            2     3                                Mario Kart Wii      Wii   
3            3     4                             Wii Sports Resort      Wii   
4            4     5                      Pokemon Red/Pokemon Blue       GB   
5            5     6                                        Tetris       GB   
6            6     7                         New Super Mario Bros.       DS   
7            7     8                                      Wii Play      Wii   
8            8     9                     New Super Mario Bros. Wii      Wii   
9            9    10                                     Duck Hunt      NES   
10          10    11                                    Nintendogs       DS   
11          11    12                                 Mario Kart DS       DS   
12          12    13                   Pokemon Gold/Pokemon Silver       GB   
13          13    14                                       Wii Fit      Wii   
14          14    15                                  Wii Fit Plus      Wii   
15          15    16                            Kinect Adventures!     X360   
16          16    17                            Grand Theft Auto V      PS3   
17          17    18                 Grand Theft Auto: San Andreas      PS2   
18          18    19                             Super Mario World     SNES   
19          19    20  Brain Age: Train Your Brain in Minutes a Day       DS   
20          20    21                 Pokemon Diamond/Pokemon Pearl       DS   
21          21    22                              Super Mario Land       GB   
22          22    23                           Super Mario Bros. 3      NES   
23          23    24                            Grand Theft Auto V     X360   
24          24    25                   Grand Theft Auto: Vice City      PS2   
25          25    26                 Pokemon Ruby/Pokemon Sapphire      GBA   
26          26    27                   Pokemon Black/Pokemon White       DS   
27          27    28   Brain Age 2: More Training in Minutes a Day       DS   
28          28    29                        Gran Turismo 3: A-Spec      PS2   
29          29    30                Call of Duty: Modern Warfare 3     X360   
30          30    31       Pokémon Yellow: Special Pikachu Edition       GB   
31          31    32                       Call of Duty: Black Ops     X360   
32          32    33                           Pokemon X/Pokemon Y      3DS   
33          33    34                     Call of Duty: Black Ops 3      PS4   
34          34    35                    Call of Duty: Black Ops II      PS3   
35          35    36                    Call of Duty: Black Ops II     X360   
36          36    37                Call of Duty: Modern Warfare 2     X360   
37          37    38                Call of Duty: Modern Warfare 3      PS3   
38          38    39                          Grand Theft Auto III      PS2   
39          39    40                       Super Smash Bros. Brawl      Wii   
40          40    41                       Call of Duty: Black Ops      PS3   
41          41    42                   Animal Crossing: Wild World       DS   
42          42    43                                  Mario Kart 7      3DS   
43          43    44                                        Halo 3     X360   
44          44    45                            Grand Theft Auto V      PS4   
45          45    46          Pokemon HeartGold/Pokemon SoulSilver       DS   
46          46    47                                Super Mario 64      N64   
47          47    48                                Gran Turismo 4      PS2   
48          48    49                            Super Mario Galaxy      Wii   
49          49    50     Pokemon Omega Ruby/Pokemon Al

## Top 3 Genre of Games that were sold the most

In [29]:
Sales["Genre"].value_counts().head(3)

Genre
Action    3253
Sports    2304
Misc      1710
Name: count, dtype: int64

According to the results The Most Sold 'Genre' is 'Action' with a massive lead with a difference of 949 with the 2nd place 'Sports' and on the 3d position is Miscelanious

In what year the Highest number of Action Gmes were sold?

In [78]:
Sales[Sales['Genre']=='Action'].groupby(by=['Year'])['Year'].value_counts().sort_values(ascending=False).head(5)

Year
2009.0    272
2012.0    266
2015.0    255
2011.0    239
2010.0    226
Name: count, dtype: int64

Action Games were sold most in year 2009 and since then the Trend of action Genre has gone up as compared to past.

WHat was the Name of the Game highest sold all over the world in 'Action Genre'

In [106]:
Sales[Sales['Genre']=='Action'].sort_values(by=['Global_Sales'], ascending=False).head(5)

Unnamed: 0  Rank                           Name Platform    Year   Genre  \
16          16    17             Grand Theft Auto V      PS3  2013.0  Action   
17          17    18  Grand Theft Auto: San Andreas      PS2  2004.0  Action   
23          23    24             Grand Theft Auto V     X360  2013.0  Action   
24          24    25    Grand Theft Auto: Vice City      PS2  2002.0  Action   
38          38    39           Grand Theft Auto III      PS2  2001.0  Action   

               Publisher  NA_Sales  EU_Sales  JP_Sales  Other_Sales  \
16  Take-Two Interactive      7.01      9.27      0.97         4.14   
17  Take-Two Interactive      9.43      0.40      0.41        10.57   
23  Take-Two Interactive      9.63      5.31      0.06         1.38   
24  Take-Two Interactive      8.41      5.49      0.47         1.78   
38  Take-Two Interactive      6.99      4.51      0.30         1.30   

    Global_Sales  
16         21.40  
17         20.81  
23         16.38  
24         16.15  
38         13.10

Video Games Sold globally HIghest in Action Category is "Grand Theft Auto V"

## Which Year had highest sales in Video game market

In [41]:
Sales["Year"].value_counts().head(12)

Year
2009.0    1431
2008.0    1428
2010.0    1259
2007.0    1202
2011.0    1139
2006.0    1008
2005.0     941
2002.0     829
2003.0     775
2004.0     763
2012.0     657
2015.0     614
Name: count, dtype: int64

We can observe the era of Gaming reached its peak during 2009.

The Sale of Games showed an exponential growth around 2005-2006 and soon fell rapidly.

What kind of Games were sold in 2009 the highest?

In [63]:
Sales[Sales['Year']==2009].groupby(by=['Genre'])['Genre'].value_counts().sort_values(ascending=False).head(5)

Genre
Action        272
Misc          207
Sports        184
Adventure     141
Simulation    123
Name: count, dtype: int64

Highest Genre of Games Sold in 2009 was 'Action'

## What kind of Platforms were used to buy Games the most

In [33]:
Sales["Platform"].value_counts().head(5)

Platform
DS      2133
PS2     2127
PS3     1304
Wii     1290
X360    1235
Name: count, dtype: int64

We can observe that Nintendo DS was the PLatform with the most selling games till date.
Playstation-2 is also not behind and is nearly equal to Nintendo DS and they can be considered nearly tied on the first place.

In  what Year did the 'DS' Games were sold the most?

In [72]:
Sales[Sales['Platform']=='DS'].groupby(by=['Year'])['Year'].value_counts().sort_values(ascending=False).head(5)

Year
2008.0    492
2009.0    405
2007.0    377
2010.0    326
2006.0    202
Name: count, dtype: int64

In 2008 Highest Number Of 'DS' (492) copies of games were sold

What kind of Games were being played on 'DS' the most?

In [80]:
Sales[Sales['Platform']=='DS'].groupby(by=['Genre'])['Genre'].value_counts().sort_values(ascending=False).head(5)

Genre
Misc          389
Action        336
Simulation    280
Adventure     238
Puzzle        236
Name: count, dtype: int64

As we Can see the HIghest Type of Games played on DS Platform were of Miscelaneous Types and following with Action